package.json

```diff
@@ -1,60 +1,72 @@
 {
-  "name": "app-backend",
-  "version": "0.1.0",
-  "description": "The backend API for the Coding Garden community app.",
-  "main": "src/index.js",
-  "jest": {
-    "testEnvironment": "node",
-    "preset": "jest-puppeteer"
-  },
-  "scripts": {
-    "start": "cross-env NODE_ENV=production node src/index.js",
-    "dev": "cross-env NODE_ENV=development nodemon src/index.js",
-    "test": "cross-env NODE_ENV=test jest --verbose --detectOpenHandles --forceExit",
-    "lint": "eslint . --ext .js",
-    "seed": "node src/tasks/seed/localDevSeed.js",
-    "deploy": "./deploy.sh",
-    "format": "prettier --write src/**/*.js",
-    "buildAPIDoc": "swagger-ui-watcher ./Swag/MainSwag.yaml --bundle=./docs/APIs.json"
-  },
-  "repository": "https://github.com/CodingGardenCommunity/app-backend.git",
-  "author": "Coding Garden Community App",
-  "license": "MIT",
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn run lint && yarn run format && yarn run test"
+    "name": "app-backend",
+    "version": "0.1.0",
+    "description": "The backend API for the Coding Garden community app.",
+    "main": "src/index.js",
+    "jest": {
+        "testEnvironment": "node",
+        "preset": "jest-puppeteer",
+        "collectCoverageFrom": [
+            "!**node_modules**",
+            "src/api/**/*.js"
+        ],
+        "coverageThreshold": {
+            "global": {
```

Comment:
Thresholds can be higher than 70, as long as we make sure that future contributions are tested/covered.